### PR TITLE
Display Pop up with selected item's details

### PR DESCRIPTION
**Pull request by jaflih**
### Display Pop up with selected item's **details**

- Added HTML element to open and close the popup
- Added popup in index.html
- Added popup style
- Added popup JS functions
- Added API call to get one pokemon from its ID
- Updated pokemon characteristics in the popup

### CODE REVIEW

File: src/modules/api.js [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
+    const results = await this.#fetchResults();
+    results.forEach((result) => {
+      promises.push(this.#fetchPockemon(result.url));
+    });
+    const items = await Promise.all(promises);
+    return items;
+  };
+
+  getPokemonFromId = async (idPokemon) => {
+    const data = await this.#fetchPockemon(`${this.POKEMON_URL}/${idPokemon}`);
```
Comment (matovu-farid) [inline src/modules/api.js]: Please insert the last back-slash in order to fix strict CORS erros
```suggestion
    const data = await this.#fetchPockemon(`${this.POKEMON_URL}/${idPokemon}/`);
```

**Review by matovu-farid — CHANGES_REQUESTED** [review]
Great job Jihanne 👍 . You have really implemented the popup feature, however, there are a few changes to work. Please check the inline comments

File: src/modules/api.js (hunk 1 above)
Comment (jaflih) [reply src/modules/api.js]: Ok, thank you! I didn't know that before. I'll correct it

**Review by matovu-farid — APPROVED** [review]
There is nothing more to say, it is time to merge it

**Comment by jaflih** [comment]
Thank you for your review.

**Pull request merged**